api/tokens.go

```diff
@@ -145,35 +143,34 @@ func (api *API) RefreshHandler(ctx context.Context, w http.ResponseWriter, req *
 		return nil, models.NewErrorResponse(http.StatusBadRequest, nil, validationErrs...)
 	}
 
-	authInput := refreshToken.GenerateRefreshRequest(api.ClientSecret, idToken.Claims.CognitoUser, api.ClientId)
+	authInput := refreshToken.GenerateRefreshRequest(api.ClientSecret, idToken.Claims.CognitoUser, api.ClientID)
 	result, authErr := api.CognitoClient.InitiateAuth(authInput)
 
 	if authErr != nil {
 		responseErr := models.NewCognitoError(ctx, authErr, "Cognito InitiateAuth request for token refresh")
 		if responseErr.Code == models.NotAuthorisedError {
 			return nil, models.NewErrorResponse(http.StatusForbidden, nil, responseErr)
-		} else {
-			return nil, models.NewErrorResponse(http.StatusInternalServerError, nil, responseErr)
 		}
+		return nil, models.NewErrorResponse(http.StatusInternalServerError, nil, responseErr)
 	}
 
-	jsonResponse, responseErr := refreshToken.BuildSuccessfulJsonResponse(ctx, result)
+	jsonResponse, responseErr := refreshToken.BuildSuccessfulJSONResponse(ctx, result)
 	if responseErr != nil {
 		return nil, models.NewErrorResponse(http.StatusInternalServerError, nil, responseErr)
 	}
 
 	headers := map[string]string{
 		AccessTokenHeaderName: "Bearer " + *result.AuthenticationResult.AccessToken,
-		IdTokenHeaderName:     *result.AuthenticationResult.IdToken,
+		IDTokenHeaderName:     *result.AuthenticationResult.IdToken,
 	}
 
 	return models.NewSuccessResponse(jsonResponse, http.StatusCreated, headers), nil
 }
 
 // SignOutAllUsersHandler bulk refresh token invalidation for panic sign out handling
-func (api *API) SignOutAllUsersHandler(ctx context.Context, w http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
+func (api *API) SignOutAllUsersHandler(ctx context.Context, _ http.ResponseWriter, req *http.Request) (*models.SuccessResponse, *models.ErrorResponse) {
 	var (
-		userFilterString string = "status=\"Enabled\""
+		userFilterString = "status=\"Enabled\""
```

Comment:
This would be clearer using a backquote'd string literal
```suggestion
		userFilterString = `status="Enabled"`
```